src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationRepositoryTests.cs

```diff
@@ -1014,28 +951,22 @@ await using (var context = InMemoryApplicationDbContext(contextId))
             await using (var context = InMemoryApplicationDbContext(contextId))
             {
                 var repository = SetupPublicationRepository(context);
-                var viewModel = await repository.GetPublicationWithAllReleases(publication.Id);
-
-                Assert.Equal(publication.Id, viewModel.Id);
-                Assert.Equal(publication.Title, viewModel.Title);
-                Assert.Equal(publication.TopicId, viewModel.TopicId);
-                Assert.Equal(publication.Contact.ContactName, viewModel.Contact.ContactName);
-
-                Assert.Equal(2, viewModel.Releases.Count);
-                Assert.Equal(release1.Id, viewModel.Releases[1].Id);
-                Assert.Equal(release2.Id, viewModel.Releases[0].Id);
-
-                Assert.Single(viewModel.Methodologies);
-                Assert.Equal(methodologyVersion.Id, viewModel.Methodologies[0].Methodology.Id);
-                Assert.Equal(methodologyVersion.AlternativeTitle, viewModel.Methodologies[0].Methodology.Title);
-
-                Assert.Equal(3, viewModel.LegacyReleases.Count);
-                Assert.Equal("Legacy Release A", viewModel.LegacyReleases[0].Description);
-                Assert.Equal("legacy-release-a-url", viewModel.LegacyReleases[0].Url);
-                Assert.Equal("Legacy Release B", viewModel.LegacyReleases[1].Description);
-                Assert.Equal("legacy-release-b-url", viewModel.LegacyReleases[1].Url);
-                Assert.Equal("Legacy Release C", viewModel.LegacyReleases[2].Description);
-                Assert.Equal("legacy-release-c-url", viewModel.LegacyReleases[2].Url);
+                var resultPublication = await repository.GetPublicationWithAllReleases(publication.Id);
+
+                Assert.Equal(publication.Id, resultPublication.Id);
+                Assert.Equal(publication.Title, resultPublication.Title);
+                Assert.Equal(publication.TopicId, resultPublication.TopicId);
+                Assert.Equal(publication.Contact.ContactName, resultPublication.Contact.ContactName);
+
+                Assert.Equal(2, resultPublication.Releases.Count);
+
+                Assert.Single(resultPublication.Methodologies);
+
+                Assert.Equal(3, resultPublication.LegacyReleases.Count);
+
+                Assert.Equal(publication.LegacyReleases[0].Id, resultPublication.LegacyReleases[0].Id);
```

Comment:
After switching I got this failure:
```
Begin Differences (1 differences):
Types [Guid,Guid], Item Expected[0].Publication.Methodologies[0].Methodology.Versions[0].MethodologyContent.MethodologyVersionId != Actual[0].Publication.Methodologies[0].Methodology.Versions[0].MethodologyContent.MethodologyVersionId, Values (339b8c89-9b59-45b1-9a3a-6dc8e0a613a7,00000000-0000-0000-0000-000000000000)
End Differences (Maximum of 1 differences shown).
Expected: True
Actual:   False
```

I think this is due to the test data not having an Id set, rather than an actual issue. So I'll not switch to using `AssertDeppEqualTo`.